deep_qa/data/fields/text_field.py

```diff
@@ -0,0 +1,72 @@
+"""
+A ``TextField`` represents a string of text, the kind that you might want to represent with
+standard word vectors, or pass through an LSTM.
+"""
+from typing import Dict, List
+
+from overrides import overrides
+import numpy
+
+from ..token_indexers import TokenIndexer, TokenType
+from ..tokenizers import Tokenizer
+from ..vocabulary import Vocabulary
+from .field import RawField, TokenizedField, IndexedField
+
+
+
+class IndexedTextField(IndexedField):
+    def __init__(self, token_arrays: List[List[TokenType]], token_indexers: List[TokenIndexer]):
+        self._token_arrays = token_arrays
+        self._token_indexers = token_indexers
+
+    @overrides
+    def get_padding_lengths(self) -> Dict[str, int]:
+        lengths = []
+        for indexer, array in zip(self._token_indexers, self._token_arrays):
+            indexer_lengths = {}
+            token_lengths = [indexer.get_padding_lengths(token) for token in array]
+            for key in token_lengths[0]:
```

Comment:
But it's a list of dicts, so `token_lengths[0]` is a dict.  Any suggestions for how to make this more clear?  Does it need a comment?